core/src/main/java/com/segment/analytics/kotlin/core/platform/policies/CountBasedFlushPolicy.kt

```diff
@@ -0,0 +1,35 @@
+package com.segment.analytics.kotlin.core.platform.policies
+
+import com.segment.analytics.kotlin.core.BaseEvent
+
+/**
+ * A Count based Flush Policy that instructs the EventPipeline to flush at the
+ * given @param[count]. The default value is 20. @param[count] values should
+ * be >= 1 or they'll get the default value.
+ */
+class CountBasedFlushPolicy(count: Int = 20): FlushPolicy {
```

Comment:
maybe directly put `flushAt` here? to avoid confusion with the internal `count`
```suggestion
class CountBasedFlushPolicy(val flushAt: Int = 20): FlushPolicy {
```